### ISSUE TITLE
Geoserver url should be configurable to work without a port or with different domain even

### ISSUE THREAD

**Issue by raitisbe** [jirik/gspld#19]
We will have to run geoserver behind some proxy to be able to connect through port 80, but in that case in service metadata it should be printed differently (http://host/gs/geoserver) than what we are using internally (http://host:8600).
![image](https://user-images.githubusercontent.com/395514/62431002-483e5780-b72c-11e9-9d80-3011486e55ce.png)

**Comment by jirik** [comment]
You probably want to set Proxy Base URL: https://docs.geoserver.org/stable/en/user/configuration/globalsettings.html#proxy-base-url

If change it, Layman should change it too (as there is a cache, it may take 60 s to see the change in Layman, but not more).

**Comment by raitisbe** [comment]
That solves it. Thanks
